site/en/docs/privacy-sandbox/glossary/index.md

```diff
@@ -108,11 +108,21 @@ out whether new data adds to entropy. For example, knowing a person is from
 Australia doesn't reduce entropy if you already know the person is from
 Kangaroo Island.
 
-## Federated identity (also known as federated login) {: #federated-identity }
+## Federated identity, federated login {: #federated-identity }
 
 A third-party platform to allow a user to sign in to a website, without
 requiring the site to implement their own identity service.
 
+## Federated Credentials Management (FedCM) {: #fedcm }
```

Comment:
In the spec repository and within the team "Federated Credentials Management" and "Federated Credential Management API" are interchangeably used in different places, but I'll just follow what's written in the spec.
```suggestion
## Federated Credential Management API (FedCM) {: #fedcm }
```